### PR TITLE
Printable Ore Bags in Autolathe

### PR DESCRIPTION
**Pull request by Lumminal**
## About the PR

Hello, this is my first PR. I added "Ore Bag" as a recipe in autolathe.

## Why / Balance

Why; I saw it requested in an open issue.

Game Balance; It will affect game balance because it will help Salvage Specialists, who lost their ore bags, to regain access to them. It can also give other players a chance to mine with the benefits of the ore bag (because Salvagers never mine)!

## Technical details

Added Ore Bag recipe in salvage.yml that costs 5 plastic, 2 cloth and 1 steel. Takes 3 seconds to complete.
Added Ore Bag in staticrecipes in lathe.yml so that it appears in the autolathe in-game.

## Media

https://github.com/user-attachments/assets/7c7c1899-a2f1-4e8e-8627-dee6998a0b13

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

No breaking changes.

**Changelog**

🆑
- add: Ore Bags can now be printed in the autolathe.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -141,6 +141,7 @@
       - UtilityBelt
       - Fulton
       - FultonBeacon
+      - OreBag
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml]: add `# Goobstation` comment

File: Resources/Prototypes/Recipes/Lathes/salvage.yml [hunk 2]
```diff
@@ -66,3 +66,13 @@
...
+
+- type: latheRecipe
+  id: OreBag
+  result: OreBag # Goobstation
+  category: Tools
+  completetime: 3
+  materials:
+    Plastic: 500
+    Cloth: 200
+    Steel: 100
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Lathes/salvage.yml]: put it in _Goobstation folder and dont need comment then

**Comment by BombasterDS2** [comment]
Fix changelog (add "- " before "add")

**Comment by Lumminal** [comment]
Fixed. Thanks for the input

**Review by Piras314 — APPROVED** [review]

**Pull request merged**